src/model/PythonPluginInstance.cpp

```diff
@@ -114,23 +116,47 @@ namespace model {
     }
 
     bool PythonPluginInstance_Impl::setPluginClassName(const std::string& pluginClassName) {
-      path filePath = externalFile().filePath();
-      filePath = system_complete(filePath);
+      if (!findPluginClassNameInFile(pluginClassName)) {
+        LOG(Warn, "Could not find plugin class name '" << pluginClassName << "' in referenced external file.");
+        return false;
```

Comment:
My idea is that we should catch it early. We still want to recheck in the FT because in the meantime there's a possibility the user went and modified the Python file, and now it's outdated.